File: src/routes/checkout.ts
import { Router } from "express";
import { cart } from "../data/db";

const router = Router();

// CHECKOUT
router.post("/", (req, res) => {
  const total = cart.reduce((sum, item) => sum + item.price, 0);

  cart.length = 0; // clear cart

  res.json({
    message: "✅ Order placed!",
    total_paid: total
  });
});

export default router;

File: routes/checkout.py
from flask import Blueprint, jsonify
from data.db import cart

checkout_bp = Blueprint("checkout", __name__)

@checkout_bp.route("/", methods=["POST"])
def checkout():
    total = sum(item["price"] for item in cart)
    cart.clear()

    return jsonify({
        "message": "✅ Order placed!",
        "total_paid": total
    })
